### DISCUSSION TITLE
autogain in convolver vs autogain

### DISCUSSION THREAD

**Discussion by IcyAlmond** [wwmm/easyeffects#2078] [Q&A] [answered]
Is there a difference between the "autogain" button in the convolver effect and the standalone "autogain" effect? 
What does the convolver one actually do? it has no options compared to the standalone "autogain" effect.

**Comment by wwmm** [comment, marked as answer]
They do completely different things. In the convolver the "autogain" calculation is trying to compensate for the fact impulse response files may have different volumes. It is a gain correction applied to the impulse response and not to the audio signal. In some cases the automatic correct works fine on others not so much.

The autogain plugin on the other hand continually updates the audio signal gain in a way that the perceived loudness remains mostly the same.
